qlib/finco/task.py

```diff
@@ -246,21 +275,43 @@ def execute(self):
         if confirm is False:
             return []
 
-        model = init_instance_by_config(workflow["task"]["model"])
-        dataset = init_instance_by_config(workflow["task"]["dataset"])
-
-        with R.start(experiment_name="finCo"):
-            model.fit(dataset)
-            R.save_objects(trained_model=model)
+        model = self._context_manager.get_context("model")
+        dataset = self._context_manager.get_context("dataset")
 
+        with R.start(experiment_name="finCo", resume=True):
             # prediction
             recorder = R.get_recorder()
             sr = SignalRecord(model, dataset, recorder)
             sr.generate()
 
-        self._context_manager.set_context("model", model)
-        self._context_manager.set_context("dataset", dataset)
-        self._context_manager.set_context("recorder", recorder)
+        return [BackTestTask()]
+
+
+class BackTestTask(Task):
```

Comment:
I think backtest is done in qrun in CMDTask.
What we need to do is to analysis it further.